Outlines/ens-beginners-guide.md

```diff
@@ -0,0 +1,27 @@
+# ENS: Beginner's Guide
```

Comment:
@darrenlangley Let's make sure to start with the problem, which is something like… Highly paraphrased of course:

Ethereum addresses are long, unwieldy, and hard to remember. ENS is like DNS, in that it maps a memorable shortcut to that address.

That is what you have said in the points below, but let's make the problem we are showing people how to solve, the first and foremost point.

Reply:
Thanks will do.